docs/auth.md

```diff
@@ -175,6 +175,32 @@ generated containing the credentials configured in the `Secret`, and these
 credentials are then used to authenticate when retrieving any
 `PipelineResources`.
 
+## GitHub Apps
+
+[GitHub Apps](https://developer.github.com/apps/about-apps/) are the preferred way for automated operations say from
+within a pipeline to interact with GitHub, rather than managing a 'bot' user and personal access token.
+
+A GitHub App is installed into a GitHub Organisation and an API is used to retrieve a token that can then be used to
```

Comment:
nit: GitHub Apps can be installed on orgs or users. We should clean up the docs/examples to reflect this.

Reply:
good call - fixed